index.js

```diff
@@ -31,21 +30,38 @@ module.exports = {
   name: 'tribes2',
   manifest: {
     create: 'async',
+    publish: 'async',
     list: 'source',
     addMembers: 'async',
     start: 'async',
   },
   init(ssb, config) {
     const addGroupTangle = AddGroupTangle(ssb)
 
+    function findOrCreateGroupFeed(input, cb) {
+      const groupKey = Buffer.isBuffer(input) ? new SecretKey(input) : input
+
+      const recps = [
+        { key: groupKey.toBuffer(), scheme: keySchemes.private_group },
+      ]
+
+      const groupFeedDetails = {
+        purpose: groupKey.toString(),
+        feedFormat: 'classic',
+        recps,
+        encryptionFormat: 'box2',
+      }
+
+      ssb.metafeeds.findOrCreate(groupFeedDetails, (err, groupFeed) => {
+        if (err) return cb(err)
+        // ssb.box2.addKeypair(groupFeed.keys)
```

Comment:
?

Reply:
I thought I had deleted it, but apparently I commented it. Will delete.